src/pages/Account/Profile.tsx

```diff
@@ -33,19 +32,14 @@ export interface ProfileFormData {
   companyDepartment?: string;
 }
 
-interface PropsType {
-  location?: RouterLocation;
-}
-
-const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
+export const Profile: React.FC = () => {
   const history = useHistory();
-  const { user } = useUser();
-  const venueName = useVenueId();
-  const venueId =
-    venueName ??
-    getQueryParameters(window.location.search)?.venueId?.toString() ??
-    DEFAULT_VENUE;
-  const { returnUrl } = getQueryParameters(window.location.search);
+  const { user, userWithId } = useUser();
+
+  const venueId = useVenueId() ?? DEFAULT_VENUE;
```

Comment:
What's the logic behind this change?

Reply:
Why does it have `DEFAULT_VENUE`?

Reply:
`useVenueId` was enhanced in https://github.com/sparkletown/sparkle/pull/1754 to handle what was previously being done here

Reply:
> Why does it have DEFAULT_VENUE?

It had `DEFAULT_VENUE` previously? I assume because legacy tech debt reasons maybe? Not sure, didn't look into the logic and didn't want to change it.